test/mason/EXECENV

```diff
@@ -7,6 +7,6 @@ echo "PATH=\$CHPL_HOME/bin/$bin_subdir:\$PATH"
 # Mason envs
 echo "MASON_HOME=\$PWD/mason_home"
 # Note, this registry should only ever be used by mason update tests
-echo "MASON_REGISTRY=registry|https://github.com/chapel-lang/mason-registry"
+echo "MASON_REGISTRY=mason-registry|https://github.com/chapel-lang/mason-registry"
```

Comment:
Most of the failures I am seeing are caused by this change. Many local `mason_home` directories contain a `registry` directory instead of `mason-registry`, so we'll need to update those too:

```
[Error matching program output for mason/masonUpdateTest (execopts: 2)]
[Error matching program output for mason/chplVersion/mason-chpl-version-update (execopts: 1)]
[Error matching program output for mason/chplVersion/mason-chpl-version-update (execopts: 2)]
[Error matching program output for mason/chplVersion/mason-chpl-version-update (execopts: 15)]
[Error matching program output for mason/chplVersion/mason-chpl-version-update (execopts: 16)]
[Error matching program output for mason/chplVersion/mason-chpl-version-update (execopts: 18)]
[Error matching program output for mason/mason-example/mason-example]
[Error matching program output for mason/mason-example-with-opts/mason-example]
[Error matching program output for mason/mason-test/mason-test]
[Error matching program output for mason/publish-tests/noFork]
[Error matching program output for mason/search/mason-search (execopts: 2)]
[Error matching program output for mason/search/mason-search (execopts: 3)]
[Error matching program output for mason/search/mason-search (execopts: 4)]
[Error matching program output for mason/search/mason-search (execopts: 5)]
[Error matching program output for mason/search/mason-search (execopts: 6)]
[Error matching program output for mason/search/mason-search (execopts: 7)]
[Error matching program output for mason/search/mason-search (execopts: 8)]
[Error matching program output for mason/search/mason-search (execopts: 9)]
[Error matching program output for mason/search/mason-search (execopts: 10)]
```